erpnext/erpnext_integrations/doctype/plaid_settings/plaid_settings.py

```diff
@@ -226,7 +226,7 @@ def new_bank_transaction(transaction):
 	try:
 		tags += transaction["category"]
 		tags += ["Plaid Cat. {}".format(transaction["category_id"])]
-	except KeyError:
+	except (KeyError, TypeError):
 		pass
```

Comment:
I don't think we need a `try-except` here

```suggestion
	if transaction.get("category"):
		tags.extend(transaction.get("category"))
	if transaction.get("category_id"):
		tags.append(f"Plaid Cat. {transaction.get("category_id")}")
```

Reply:
Hi Alchez. 

Fine by me. 

@deepeshgarg007 do youagree?